graphql/src/execution/execution.rs

```diff
@@ -717,9 +723,9 @@ fn merge_selection_sets(fields: Vec<&q::Field>) -> q::SelectionSet {
 /// Coerces argument values into GraphQL values.
 pub fn coerce_argument_values<'a, R1, R2>(
     ctx: ExecutionContext<'a, R1, R2>,
-    object_type: &'a s::ObjectType,
-    field: &'a q::Field,
-) -> Result<HashMap<&'a q::Name, q::Value>, Vec<QueryExecutionError>>
+    object_type: &s::ObjectType,
+    field: &q::Field,
+) -> Result<HashMap<q::Name, q::Value>, Vec<QueryExecutionError>>
```

Comment:
What's the reason for cloning the names here rather than referencing them? Since this is only about arguments (and there won't be that many arguments in any given query) this is probably ok from a performance perspective, although I generally tried to clone as little as possible in the execution.

Reply:
Yes this is a small pessimization. Thinking about it again, I can avoid this by using`lazy_static`, I'll try that.